src/utils/style-formatter.js

```diff
@@ -42,6 +42,14 @@ const getColor = (
   return !resolvedColor || resolvedColor === 'none' ? defaultColor : resolvedColor;
 };
 
+export function measureText(text, font) {
+  const canvas = measureText.canvas || (measureText.canvas = document.createElement('canvas'));
+  const context = canvas.getContext('2d');
+  context.font = font;
+  const metrics = context.measureText(text);
+  return metrics;
+}
```

Comment:
Not really sure we want this. It will allow in the feature to calculate this with different fonts, but right we are only able to use one specific font here.